### PR TITLE
Automation of Pipeline Metrics

### PR DESCRIPTION
**Pull request by hemantsaini-7**
**Description:**

**Jira Id:** [ODC-6580](https://issues.redhat.com/browse/ODC-6580)

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [ ] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
oc login -u kubeadmin -p $(cat /path/to/install-dir/auth/kubeadmin-password)
source ./contrib/oc-environment.sh
./bin/bridge

cd frontend
oc login ...
yarn run test-cypress-pipelines
```

**Screen shots**:
image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by hemantsaini-7** [comment]
I have rebased and made the changes, please review @jrichter1

**Comment by jrichter1** [comment]
/lgtm

will also need an approval from a frontend owner because of the component change
@jerolimov can you add your approval?

/assign @jerolimov

File: frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipeline-metrics.ts [hunk 1]
```diff
@@ -43,3 +43,7 @@ Then(
       .should('contain.text', pipelineDetailsText.metrics.graphs.taskRunDuration);
   },
 );
+
+When('user can see message "No datapoints found" inside graphs', () => {
+  cy.get('[data-test="datapoints-msg"]').should('include.text', 'No datapoints found.');
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipeline-metrics.ts:48]: nit: use `byTestID`
```suggestion
  cy.byTestID("datapoints-msg").should('include.text', 'No datapoints found.');
```

**Review by karthikjeeyar — APPROVED** [review]

**Review by jerolimov — APPROVED** [review]
/approve

**Comment by jerolimov** [comment]
@hemantsaini-7 @jrichter1 Cool, can you both apply the other labels needed to get this merged?

**Comment by jrichter1** [comment]
Applying the labels, since we don't have a bz
/label px-approved
/label docs-approved
/label qe-approved

**Pull request merged**